### DISCUSSION TITLE
Flushing podman logs

### DISCUSSION THREAD

**Discussion by f1yn** [containers/podman#17811] [Q&A] [answered]
I have several dozen pods running at any given time. As my usage has grown, I've noticed that the usability of `podman logs` become unbearable as logs are usually slow to rotate, even when `containers.conf` has a smaller log size. This means my terminal gets slammed by days (or even weeks) worth of logs.

There's several solution to this for docker, but they don't translate well to podman. How can I manually flush logs in podman, either totally or on a per container basis? Has anyone figured this out?

**Comment by rhatdan** [comment]
What log driver are you using?  I would figure journald would not have this issue.  You can also turn off logging, for containers/pods that you don't want the logs for.

**Comment by f1yn** [comment]
Hey @rhatdan,

I haven't set any other log driver, so I'm assuming it'd just be `journald`. Unfortunately for auditing purposes (since some of these containers are connected to a VPC), I don't feel comfortable selectively disabling logs on a per-container basis.

It would appear it's `journald` (like I suspected):

```
               "LogConfig": {
                    "Type": "journald",
                    "Config": null,
                    "Path": "",
                    "Tag": "",
                    "Size": "0B"
               },
```

**Reply by f1yn** [reply]
Is there an explicit difference between configuring the log size via `containers.conf` vs `--log-opt max-size`?

**Comment by rhatdan** [comment]
No idea.
@nalind @Luap99 Any idea?

**Comment by f1yn** [comment, marked as answer]
Actually, turns out I made a mistake somewhere. I recently did a Fedora Server upgrade, which is around where the problem started.

During the upgrade, it would seem like `/usr/share/containers/containers.conf` was reset to it's default value. Not ideal, but that would explain why this suddenly started happening.

**Reply by f1yn** [reply]
Since I run everything rootless, I'll just opt to set the local user `containers.conf` instead 😂

**Reply by f1yn** [reply]
Thanks for your time @rhatdan

**Comment by rhatdan** [comment]
If you want to customize containers.conf system wide, you should do it in /etc/containers/containers.conf
